examples/evaluator/golang/simple/evaluate/evaluator.go

```diff
@@ -15,40 +15,78 @@
 package evaluate
 
 import (
-	"open-match.dev/open-match/examples"
+	"math"
+	"sort"
+
+	"github.com/golang/protobuf/ptypes"
+	"github.com/sirupsen/logrus"
 	harness "open-match.dev/open-match/pkg/harness/evaluator/golang"
 	"open-match.dev/open-match/pkg/pb"
 )
 
 // Evaluate is where your custom evaluation logic lives.
 // This sample evaluator sorts and deduplicates the input matches.
 func Evaluate(p *harness.EvaluatorParams) ([]*pb.Match, error) {
-	scoreInDescendingOrder := func(a, b *pb.Match) bool {
-		return a.GetProperties().GetFields()[examples.MatchScore].GetNumberValue() > b.GetProperties().GetFields()[examples.MatchScore].GetNumberValue()
+	matches := make(matcheExts, 0, len(p.Matches))
+	for _, match := range p.Matches {
+		// Evaluation criteria is optional, but sort it lower than any matches which
+		// provided criteria.
+		ext := &pb.DefaultEvaluationCriteria{
+			Score: math.Inf(-1),
+		}
+		if match.Extension != nil {
```

Comment:
A warning log if this is nil? Basically that match is completely ignored - so some trace in logs will be useful to debug.

Reply:
_This is desired behavior, as for someone who isn't aware of the evaluator, it just works_

Thanks for clarifying the behavior - I agree the behavior is appropriate. I think adding more to the logs is helpful - leaves a trace of what happens. If somoene does not populate any score in MMF because they just didnt know that was a thing - I think it is reasonable (and a very critical debugging piece) for logs to give a hint.

Reply:
Ok, I don't want to spam the logs too much, and it definitely shouldn't be a warning because nothing is going wrong.  Adding info logs once per evaluate call.